static/js/services/contact-schema.js

```diff
@@ -259,9 +200,7 @@ angular.module('inboxServices').service('ContactSchema',
           });
         });
         return schema;
-      },
-      // Used by Enketo.
-      validate: validateSchema,
+      }
     };
   }
 );
```

Comment:
I didn't put too much effort into cleaning up this file, mostly just to make it a little bit more understandable.

This entire file could be a giant constant, no code is actually needed. However, I figure our goal is to delete the entire thing anyway, so I didn't go that far.